flipperdevices/flipperzero-firmware
# Fix GUI freeze after NFC classic read

# What's new

- Add osDelay to NFC worker to allow task switch
- Optimize RPC version characteristic

# Verification 

- Verify GUI don't freeze after Mifare Classic read
- Verify RPC version is correctly read from BLE RPC Version characteristic

# Checklist (For Reviewer)
